takeshape/takeshape-samples
# Fix regex with invalid unicode range

# Bug Fix
```
new RegExp('[\\w-_]+', 'u');
```
throws
```
Uncaught SyntaxError: Invalid regular expression: /[\w-_]+/: Invalid character class
```

TakeShape now expects patterns to valid unicode. Using this pattern will break or disable validation on the latest version of TakeShape.

### Test Plan (Steps to test):

1.  Import the shape-blog pattern using the url import
1.  Navigate to a post
1.  verify slug regex validation works

### Checklist:
